zkchannels-crypto/src/proofs/range.rs

```diff
@@ -42,6 +42,15 @@ pub struct RangeConstraintParameters {
     public_key: PublicKey<1>,
 }
 
+impl ChallengeInput for RangeConstraintParameters {
+    fn consume(&self, builder: &mut ChallengeBuilder) {
+        for digit_signature in self.digit_signatures.iter() {
+            builder.consume(digit_signature);
+        }
+        builder.consume(&self.public_key);
+    }
+}
+
```

Comment:
Good catch. Looks like before, we weren't integrating the signatures at all. Fixed this to use the whole range constraint parameters, instead of just the public key.